disorder-corpus/src/Disorder/Corpus.hs

```diff
@@ -357,3 +361,180 @@ boats = [
   , "tugboat"
   , "yacht"
   ]
+
+newtype Glass a =
+  Glass a
+  deriving (Eq, Ord, Read, Show, Generic, Data, Typeable)
+
+unGlass :: Glass a -> a
+unGlass (Glass x) =
+  x
+
+instance (Eq a, IsString a, Arbitrary a) => Arbitrary (Glass a) where
+  arbitrary =
+    genCorpus glass Glass arbitrary
+  shrink =
+    shrinkCorpus glass Glass unGlass shrink
+
+-- | How to say "I can eat glass, and it doesn't hurt me." in a few different
+--   languages.
+--
+--   From: http://kermitproject.org/utf8.html
+--
+glass :: IsString a => [a]
+glass = [
+    "काचं शक्नोम्यत्तुम् । नोपहिनस्ति माम् ॥" -- Sanskrit
+  , "kācaṃ śaknomyattum; nopahinasti mām." -- Sanskrit (standard transcription)
+  , "ὕαλον ϕαγεῖν δύναμαι· τοῦτο οὔ με βλάπτει." -- Classical Greek
+  , "Μπορώ να φάω σπασμένα γυαλιά χωρίς να πάθω τίποτα." -- Greek (monotonic)
+  , "Μπορῶ νὰ φάω σπασμένα γυαλιὰ χωρὶς νὰ πάθω τίποτα. " -- Greek (polytonic)
+  , "Vitrum edere possum; mihi non nocet." -- Latin
+  , "Je puis mangier del voirre. Ne me nuit." -- Old French
+  , "Je peux manger du verre, ça ne me fait pas mal." -- French
+  , "Pòdi manjar de veire, me nafrariá pas." -- Provençal / Occitan
+  , "J'peux manger d'la vitre, ça m'fa pas mal." -- Québécois
+  , "Dji pou magnî do vêre, çoula m' freut nén må. " -- Walloon
+  , "Ch'peux mingi du verre, cha m'foé mie n'ma. " -- Picard
+  , "Mwen kap manje vè, li pa blese'm." -- Kreyòl Ayisyen (Haitï)
+  , "Kristala jan dezaket, ez dit minik ematen." -- Basque
+  , "Puc menjar vidre, que no em fa mal." -- Catalan / Català
+  , "Puedo comer vidrio, no me hace daño." -- Spanish
+  , "Puedo minchar beire, no me'n fa mal . " -- Aragonés
+  , "Eu podo xantar cristais e non cortarme." -- Galician
+  , "Posso comer vidro, não me faz mal." -- European Portuguese
+  , "Posso comer vidro, não me machuca." -- Brazilian Portuguese
+  , "M' podê cumê vidru, ca ta maguâ-m '." --Caboverdiano/Kabuverdianu (Cape Verde)
+  , "Ami por kome glas anto e no ta hasimi daño." -- Papiamentu
+  , "Posso mangiare il vetro e non mi fa male." -- Italian
+  , "Sôn bôn de magnà el véder, el me fa minga mal." -- Milanese
+  , "Me posso magna' er vetro, e nun me fa male." -- Roman
+  , "M' pozz magna' o'vetr, e nun m' fa mal." -- Napoletano
+  , "Mi posso magnare el vetro, no'l me fa mae." -- Venetian
+  , "Pòsso mangiâ o veddro e o no me fà mâ." -- Zeneise (Genovese)
+  , "Puotsu mangiari u vitru, nun mi fa mali. " -- Sicilian
+  , "Jau sai mangiar vaider, senza che quai fa donn a mai. " -- Romansch (Grischun)
+  , "Pot să mănânc sticlă și ea nu mă rănește." -- Romanian
+  , "Mi povas manĝi vitron, ĝi ne damaĝas min. " -- Esperanto
+  , "Mý a yl dybry gwéder hag éf ny wra ow ankenya." -- Cornish
+  , "Dw i'n gallu bwyta gwydr, 'dyw e ddim yn gwneud dolur i mi." -- Welsh
+  , "Foddym gee glonney agh cha jean eh gortaghey mee." -- Manx Gaelic
+  , "᚛᚛ᚉᚑᚅᚔᚉᚉᚔᚋ ᚔᚈᚔ ᚍᚂᚐᚅᚑ ᚅᚔᚋᚌᚓᚅᚐ᚜" -- Old Irish (Ogham)
```

Comment:
They're designed as a browser torture test. If you're seeing boxes, you're hurting your browser and should stop